config/secrets.yml

```diff
@@ -10,11 +10,13 @@
 # Make sure the secrets in this file are kept private
 # if you're sharing your code publicly.
 
-production:
-  secret_key_base: <%= ENV["SECRET_KEY_BASE"] %>
-
 development:
-  secret_key_base: 526701f5b66b8163622e6358f6989f8ee10b27b41dcbdf587f4075208d3efe9b40da8ce5e82bf1e04937c1e4f89fcd3630f496aefcaa70138113c228e0af8d67
+  secret_key_base: ebc040546747a3f1ccdc345ba503efc5e6dff153319f34ca5cb30b500ee513bfaed4c6f89551f9da7eb2df728e83938eebf813071b0d03b687c0d5deb1a78db6
```

Comment:
I think it's not a good practice to expose the tokens, even for development or test environments